api/ScreenOrientation.json

```diff
@@ -114,9 +114,17 @@
             "firefox": {
               "version_added": "43"
             },
-            "firefox_android": {
-              "version_added": "43"
-            },
+            "firefox_android": [
+              {
+                "version_added": "79",
+                "partial_implementation": true,
+                "notes": "The API exists but does not lock the screen orientation."
+              },
+              {
+                "version_added": "43",
+                "version_removed": "79"
```

Comment:
Note, Fennec last version was FF68. The next version is FF79, which uses Fenix and does not support the API on android. So FF79 is the version in which support was removed.